File: Java/Akademik.java
public class Akademik extends Human{
    private String kampus;
    private String email;

    public Akademik(){
        this.kampus = "";
        this.email = "";
    }

    public Akademik(String kampus, String email){
        this.kampus = kampus;
        this.email = email;
    }

    /* Setter */
    public void setKampus(String kampus){
        this.kampus = kampus;
    }

    public void setEmail(String email){
        this.email = email;
    }

    /* Setter */
    public String getKampus(){
        return this.kampus;
    }

    public String getEmail(){
        return this.email;
    }



}

File: C++/Akademik.cpp
class Akademik : public Human{
    private :
        string kampus;
        string email;

    public :

        Akademik(){
        }

        /* Setter and Getter Kampus*/
        void setKampus(string kampus){
            this->kampus = kampus;
        }

        string getKampus(){
            return this->kampus;
        }

        /* Setter and Getter Email*/
        void setEmail(string email){
            this->email = email;
        }

        string getEmail(){
            return this->email;
        }

        ~Akademik(){

        }
};